ecs-cli/modules/utils/utils.go

```diff
@@ -130,6 +135,7 @@ func ParseLoadBalancers(flagValues []string) ([]*ecs.LoadBalancer, error) {
 			if len(pair) > 2 {
```

Comment:
Would it be better to have like this
```suggestion
			if len(pair) != 2 {
			  return nil, fmt.Errorf("Only include one = to indicate your value in your %s", key)
			}
			key, val := pair[0], pair[1]
```
so that it is able to catch the case when input does not have any "=" and prevent the golang panic.

Reply:
I like this! One issue is where the key is not initialized before. So I could either do pair[0] for the error message or not include the %s and just display an error message.